app/src/main/java/org/simple/clinic/summary/teleconsultation/api/TeleconsultationsResponse.kt

```diff
@@ -5,6 +5,12 @@ import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
 data class TeleconsultationsResponse(
+    @Deprecated(
+        message = "Use teleconsultationPhoneNumbers to get list of all MO phone numbers",
+        replaceWith = ReplaceWith("teleconsultationPhoneNumbers", "org.simple.clinic.summary.teleconsultation.api")
+    )
     @Json(name = "teleconsultation_phone_number")
-    val teleconsultationPhoneNumber: String?
+    val teleconsultationPhoneNumber: String?,
+    @Json(name = "teleconsultation_phone_numbers")
```

Comment:
Can we add a line break here to separate the two properties and make it more readable?

Reply:
Done